workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.ts

```diff
@@ -557,7 +557,9 @@ export default class DynamoDBService {
             item: marshall(request.item, { removeUndefinedValues: true }),
             conditionExpression: request.conditionExpression,
             expressionAttributeNames: request.expressionAttributeNames,
-            expressionAttributeValues: marshall(request.expressionAttributeValues)
+            expressionAttributeValues: request.expressionAttributeValues
```

Comment:
Why did we need this change?

Reply:
if expressionAttributeValues is undefined, it will marshall an undefined value -> throwing an error